examples/30_extended/plot_svm_hyperparameters_tutorial.py

```diff
@@ -0,0 +1,74 @@
+"""
+================================
+Plotting hyperparameter surfaces
+================================
+"""
+import openml
+import numpy as np
+
+# Choose an SVM flow, for example 8353, and a task.
+df = openml.evaluations.list_evaluations_setups(
+    function='predictive_accuracy',
+    flow=[8353],
+    task=[6],
+    output_format='dataframe',
+    parameters_in_separate_columns=True,
+)
+hyperparameters = ['sklearn.svm.classes.SVC(16)_C', 'sklearn.svm.classes.SVC(16)_gamma']
```

Comment:
How about printing the column names here to show that `sklearn.svm.classes.SVC(16)_C` & `sklearn.svm.classes.SVC(16)_gamma` or a check whether it is in the colnames list with the _in_ operator?